exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -322,6 +332,25 @@ func (b *Bitfinex) UpdateTradablePairs(ctx context.Context, forceUpdate bool) er
 	return b.EnsureOnePairEnabled()
 }
 
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (b *Bitfinex) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
+	}
+	pairs, err := b.GetSiteInfoConfigData(ctx, a)
+	if err != nil {
+		return err
+	}
+	limits := make([]order.MinMaxLevel, 0, len(pairs))
+	for i := range pairs {
+		limits = append(limits, pairs[i])
```

Comment:
If you end up changing the response to a slice instead of a map, you won't even need the limits loop

Reply:
that is done